src/sqlparser/src/ast/ddl.rs

```diff
@@ -830,6 +833,10 @@ impl fmt::Display for ColumnOption {
             Check(expr) => write!(f, "CHECK ({})", expr),
             DialectSpecific(val) => write!(f, "{}", display_separated(val, " ")),
             GeneratedColumns(expr) => write!(f, "AS {}", expr),
+            DerivedFromAdditionalColumns => {
+                // write nothing because it cannot be defined in sql
+                Ok(())
```

Comment:
When calling `ALTER TABLE [ADD | DROP] COLUMN` after dropping the connector, will the additional column be rejected again?